### PR TITLE
drainer: add read-timeout configuration explanation

### PR DESCRIPTION
**Pull request by lichunzhu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add read-timeout configuration explanation

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 1]
```diff
@@ -317,6 +317,7 @@ tbl-name = "~^a.*"
 * `port`：如果没有设置，会尝试检查环境变量 `MYSQL_PORT`，默认值为 `3306`。
 * `user`：如果没有设置，会尝试检查环境变量 `MYSQL_USER`，默认值为 `"root"`。
 * `password`：如果没有设置，会尝试检查环境变量 `MYSQL_PSWD`，默认值为 `""`。
+* `read-timeout`：指定下游数据库连接的读取超时时间，如果 drainer 在一些长耗时的 ddls 上不断失败，你可以将这个变量设置为更大的时间，默认值为 `"1m"`。
```
Comment (shichun-0415) [inline tidb-binlog/tidb-binlog-configuration-file.md:320]:
```suggestion
* `read-timeout`：指定下游数据库连接的读取超时时间，默认值为 `"1m"`，如果 drainer 在一些长耗时的 DDL 上不断失败，你可以将这个变量设置为更大的时间。
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

File: tidb-binlog/tidb-binlog-configuration-file.md (hunk 1 above)
Comment (Oreoxmt) [inline tidb-binlog/tidb-binlog-configuration-file.md:320]:
```suggestion
* `read-timeout`：指定下游数据库连接的读取超时时间，默认值为 `"1m"`。如果 drainer 在一些耗时长的的 DDL 上不断失败，你可以将这个变量设置为更大的值。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 2]
```diff
@@ -317,6 +317,7 @@ tbl-name = "~^a.*"
 * `port`：如果没有设置，会尝试检查环境变量 `MYSQL_PORT`，默认值为 `3306`。
 * `user`：如果没有设置，会尝试检查环境变量 `MYSQL_USER`，默认值为 `"root"`。
 * `password`：如果没有设置，会尝试检查环境变量 `MYSQL_PSWD`，默认值为 `""`。
+* `read-timeout`：指定下游数据库连接的读取超时时间，默认值为 `"1m"`。如果 drainer 在一些耗时长的 DDL 上不断失败，你可以将这个变量设置为更大的值。
```
Comment (lance6716) [inline tidb-binlog/tidb-binlog-configuration-file.md:320]:
```suggestion
* `read-timeout`：指定下游数据库连接的 IO 读取超时时间，默认值为 `"1m"`。如果 drainer 在一些耗时长的 DDL 上不断失败，你可以将这个变量设置为更大的值。
```

下游是“写入”的数据库，这里又提到一个“读取”可能用户会觉得奇怪。有什么表达“读取‘写入的响应’”的方法吗

**Review by lance6716 — APPROVED** [review]

File: tidb-binlog/tidb-binlog-configuration-file.md (hunk 2 above)
Comment (lichunzhu) [reply tidb-binlog/tidb-binlog-configuration-file.md:320]: LGTM

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 3]
```diff
@@ -317,6 +317,7 @@ tbl-name = "~^a.*"
 * `port`：如果没有设置，会尝试检查环境变量 `MYSQL_PORT`，默认值为 `3306`。
 * `user`：如果没有设置，会尝试检查环境变量 `MYSQL_USER`，默认值为 `"root"`。
 * `password`：如果没有设置，会尝试检查环境变量 `MYSQL_PSWD`，默认值为 `""`。
+* `read-timeout`：指定下游数据库连接的 IO 读取超时时间，默认值为 `"1m"`。如果 drainer 在一些耗时长的 DDL 上不断失败，你可以将这个变量设置为更大的值。
```
Comment (shichun-0415) [inline tidb-binlog/tidb-binlog-configuration-file.md:320]:
```suggestion
* `read-timeout`：指定下游数据库连接的 IO 读取超时时间，默认值为 `1m`。如果 drainer 在一些耗时长的 DDL 上不断失败，你可以将这个变量设置为更大的值。
```

**Pull request merged**